api/server/server.go

```diff
@@ -184,16 +178,13 @@ func NewServer(conf Config) (*Server, error) {
 		return nil, fmt.Errorf("creating ipfs client: %s", err)
 	}
 
-	lchain := lotuschain.New(c)
+	chain := filchain.New(c)
 	ms := reptop.New(rm, ai)
 
 	l := cidlogger.New(txndstr.Wrap(ds, "ffs/scheduler/logger"))
-	cs := filcold.New(ms, dm, ipfs, lchain, l)
+	cs := filcold.New(ms, dm, ipfs, chain, l)
 	hs := coreipfs.New(ipfs, l)
-	js := jstore.New(txndstr.Wrap(ds, "ffs/scheduler/jstore"))
-	as := astore.New(txndstr.Wrap(ds, "ffs/scheduler/astore"))
-	cis := cistore.New(txndstr.Wrap(ds, "ffs/scheduler/cistore"))
-	sched := scheduler.New(js, as, cis, l, hs, cs)
+	sched := scheduler.New(txndstr.Wrap(ds, "ffs/scheduler"), l, hs, cs)
```

Comment:
All the deleted stores are internal things of the `scheduler`, which now is responsible for creating them, and closing them. The `server.go` only knows about `scheduler`, don't have to worry about internal things.